handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Install metasploit
+  when: metasploit.changed
+  become: true
+  command: /tmp/msfinstall
```

Comment:
We prefer the fully-qualified collection names (FQCNs) to avoid warnings from the Ansible linter.  In this case the FQCN is `ansible.builtin.command`.

Reply:
Same as https://github.com/cisagov/ansible-role-metasploit/pull/5#discussion_r927813243, I updated `handlers/main.yml` to use the FQCN.